src/style.css

```
@@ -1 +1,63 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.logotype {
+  margin: 10px 0 10px 50px;
```

move side margins to .container
```suggestion
  margin: 10px 0;
```